lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -110,7 +110,7 @@ private function build($data) {
       'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
       'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
       'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
+      'photo' => isset($drupal_user->picture) ? $drupal_user->picture : NULL,
```

Comment:
Maybe that's something that Northstar could add when it's getting proxied. Kinda like how with Gladiator, we send the `/signups` request to Gladiator and it will convert all the NS IDs we pass to Phoenix IDs to send to Phoenix. So what could happen is NS receives the request to `/signups` proxies to Phoenix, when Phoenix sends it back to NS, it takes control and adds the NS avatar  info then, before proceeding with sending the full response back to client? Avoids having to request from NS only to then send back to NS anyways.... @DFurnes thoughts?